generator/_assets/styles/less/header.less

```diff
@@ -41,7 +41,8 @@
       flex-direction: column;
       background: #FFFFFF;
       z-index: 10;
-      width: 24.3rem;
+      width: clamp(243px, 50%, 433px);
```

Comment:
please use rem units, in our case 1rem = 10px

Reply:
fixed